CHANGELOG.rst

```diff
@@ -104,6 +104,7 @@ Fixed
   NOTE: This issue only affected installations using Python 3. (bug fix) #4832 #4834
 
   Reported by @alexku7.
+* Fix the amqp connection setup for WorkflowExecutionHandler to pass SSL params.
```

Comment:
Please include PR number as well for the fix to keep it in history.
If you want, you're very welcome to also include your name as a contributor for the fix.

See other Changelog records as example.

Reply:
I added it.

Reply:
:+1: